README.md

```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
 When it comes to equity compensation, everything depends on understanding some fundamental
 questions:
 
-- What is stock, and how do I own it?
-  What are shares?
-- Is stock the same as equity?
-- What is my equity worth?
-- If I have or will get stock at some point, can I sell it?
+- What is stock, and how does one own it?
+- How does stock relate to shares and equity?
+- What can you do with stock or stock options?
+- What kinds of companies utilize equity compensation?
 
-This requires discussing starting with the basics of how companies and stock work in the
-United States.
-If you think you know all this, you can [jump ahead](#equity-compensation-basics) to
-compensation.
+That last question is often overlooked or misunderstood, and it’s really where everything
+starts.
+
+🄳 For-profit companies that share ownership (in divestments of stock called "shares") in
+exchange for capital or labor are called
+[**stock corporations**](https://www.thebalancesmb.com/what-is-a-stock-corporation-398479). We’ll
+explain all those terms in this chapter.
+
+We begin with the basics of how companies and stock work in the United States.
+Understanding these details will help you learn the role of equity in your employment and
+its purpose in a company’s overall structure, as well as what you can do with your stock,
+how to decide what to do, and, of course, how to do it.
+
+If you think you know all about stock corporations, you can
+[jump ahead](#equity-compensation-basics) to compensation.
 
 ### What is a company?
 
 - 🄳 A **company** is a legal entity formed under corporate law for the purpose of conducting
   trade. In the United States,
   [several kinds of business entities](https://en.wikipedia.org/wiki/Types_of_business_entity#United_States)
-  may be formed, or [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)),
+  may be formed, or [**incorporated**](<https://en.wikipedia.org/wiki/Incorporation_(business)>,
   under the laws of a specific state.
-- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company that
-  aspires to grow quickly.
+- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company,
+  typically a [private company](#public-and-private-companies), that aspires to grow quickly in
+  size, revenue, and influence.
   Once a company is established in the market and successful for a while, it usually stops
   being called a startup.
-  - ☝️ The term *startup* is pretty informal.
-    It’s not a legal concept, and not everyone uses it consistently.
-    For our purposes, it’s best just to think of it as the young period of a fast-growing
-    company.
+  - ☝️ Unlike some of the legal terms we discuss, the term *startup* is informal, and not
+    everyone uses it consistently.
+    As distinguished investor [Paul Graham]\([https://en.wikipedia.org/wiki/Paul*Graham*(programmer](<https://en.wikipedia.org/wiki/Paul_Graham_(programmer>))
```

Comment:
Whole bunch of link formatting issues, likely due to Flowmark on incorrect link syntax. See another wikipedia link in the doc for an exact example.